staging/src/k8s.io/code-generator/cmd/import-boss/README.md

```diff
@@ -0,0 +1,62 @@
+## Purpose
+
+- `import-boss` enforces import restrictions in a given repository.
+
+## How does it work?
+
+- When a directory is verified, `import-boss` looks for a file called `.import-restrictions`. If this file is not found, parent directories will be recursively searched.
+  - NOTE: Adding `.import-restrictions` files does not add them to CI runs. In order for them to be added, they need to be explicitly added to `arguments.InputDirs`. `InputDirs` is defined within `import-boss/main.go`.
```

Comment:
i'd indent the second line the same as the previous one ` - When....`
and remove the `NOTE: ` prefix:

```
- When a directory is verified...
- Adding `.import-restrictions`...
```